nixos/modules/services/misc/prowlarr.nix

```diff
@@ -11,11 +11,26 @@ in
     services.prowlarr = {
       enable = mkEnableOption (lib.mdDoc "Prowlarr");
 
+      dataDir = mkOption {
+        type = types.str;
+        default = "/var/lib/prowlarr/.config/Prowlarr";
```

Comment:
Why is this different from the original default (/var/lib/prowlarr)

Reply:
Technically, it doesn't have to. It doesn't even matter... In fact, the official docs also mention `/var/lib/prowlarr` so, I am being weird here.

I have however, attempted to normalise it with the line of `sonarr` and `radarr` for convinience of navigation. For instance:

```
/var/lib/sonarr/.config/NzbDrone/config.xml
/var/lib/radarr/.config/Radarr/config.xml
/var/lib/prowlarr/config.xml
```

I'll be happy to reset the default value to the original if you'd rather not break the user experience.